mlrun/datastore/utils.py

```diff
@@ -143,12 +142,19 @@ def select_columns_generator(
 
 def _generate_sql_query_with_time_filter(
     table_name: str,
-    engine: sqlalchemy.engine.Engine,
+    engine,
```

Comment:
you might be able to use it with quotes
```suggestion
    engine: "sqlalchemy.engine.Engine",
```

Reply:
in that case you'll get flake err:
`mlrun/datastore/utils.py:145:13: F821 undefined name 'sqlalchemy'`

Reply:
I'll add `# noqa: F821`